mapsTask/js/mapController.js

```diff
@@ -0,0 +1,122 @@
+
+function initialize() {
+    initMap();
+    initAutocomplete();
+}
+
+var map;
+var geocoder;
+var marker;
+var icon;
```

Comment:
Are these variables necessary to be global? If so, okay. Provide default values, so it's clear what their purpose is and what they will contain at some point. If not, move them into the functions` scope they are used in.